server_addon/maya/server/settings/publishers.py

```diff
@@ -1371,6 +1610,56 @@ class PublishersModel(BaseSettingsModel):
     "ExtractModel": {
         "enabled": True,
         "optional": True,
-        "active": True,
+        "active": True
+    },
+    "ExtractAlembic": {
+        "enabled": True,
+        "families": [
+            "pointcache",
+            "model",
+            "vrayproxy.alembic"
+        ],
+        "attr": "",
+        "attrPrefix": "",
+        "autoSubd": False,
+        "bake_attributes": [],
+        "bake_attribute_prefixes": [],
+        "dataFormat": "ogawa",
+        "dontSkipUnwrittenFrames": False,
+        "eulerFilter": False,
+        "melPerFrameCallback": "",
+        "melPostFrameCallback": "",
+        "noNormals": False,
+        "overrides": [
+            "attr",
+            "attrPrefix",
+            "renderableOnly",
+            "userAttr",
+            "userAttrPrefix",
```

Comment:
I believe `userAttr` is new from this PR and it can be quite confusing what the difference is between `attr` and `userAttr` in Alembic's export logic. I wonder whether we should keep the `user*` attribute equivalent out of the defaults from being exposed to be more backwards compatible and also avoid confusing artists over time seeing both `attr` and `userAttr` fields in the publisher UI.

I for one here wouldn't 'quickly' know where to type the attribute I want to export:
![image](https://github.com/ynput/ayon-core/assets/2439881/2e903cc3-6660-4fab-8b12-80f74d44a445)

Is it a User attribute or a custom attribute? :)

Reply:
```suggestion
            "attr",
            "attrPrefix",
            "renderableOnly",
```